subprojects/docs/src/docs/userguide/troubleshootingDependencyResolution.adoc

```diff
@@ -15,83 +15,93 @@
+Conflicting versions of the same module should be detected and either resolved or cause an exception. If you don't use transitive dependency management, version conflicts are undetected and may influence the order of the classpath determining what version of a dependency will be selected. In short, version conflicts can cause classpath issues and are extremely hard to debug.
 
-If you haven't had to deal with the curse of conflicting versions of jars on a classpath, here is a small anecdote of the fun that awaits you. In a large project with 30 submodules, adding a dependency to a subproject changed the order of a classpath, swapping Spring 2.5 for an older 2.4 version. While the build continued to work, developers were starting to notice all sorts of surprising (and surprisingly awful) bugs in production. Worse yet, this unintentional downgrade of Spring introduced several security vulnerabilities into the system, which now required a full security audit throughout the organization.
-
-In short, version conflicts are bad, and you should manage your transitive dependencies to avoid them. You might also want to learn where conflicting versions are used and consolidate on a particular version of a dependency across your organization. With a good conflict reporting tool like Gradle, that information can be used to communicate with the entire organization and standardize on a single version. _If you think version conflicts don't happen to you, think again._ It is very common for different first-level dependencies to rely on a range of different overlapping versions for other dependencies, and the JVM doesn't yet offer an easy way to have different versions of the same jar in the classpath (see <<sub:dependency_management_and_java>>).
-
-Gradle offers the following conflict resolution strategies:
+Gradle can <<managing_transitive_dependencies,manage transitive dependencies>> for you with the help of the following conflict resolution strategies:
 
 * _Newest_: The newest version of the dependency is used. This is Gradle's default strategy, and is often an appropriate choice as long as versions are backwards-compatible.
-* _Fail_: A version conflict results in a build failure. This strategy requires all version conflicts to be resolved explicitly in the build script. See api:org.gradle.api.artifacts.ResolutionStrategy[] for details on how to explicitly choose a particular version.
+* _Fail_: A version conflict results in a build failure. This strategy requires all version conflicts to be resolved explicitly in the build script. Fore more information, see api:org.gradle.api.artifacts.ResolutionStrategy#failOnVersionConflict()[].
 
 While the strategies introduced above are usually enough to solve most conflicts, Gradle provides more fine-grained mechanisms to resolve version conflicts:
```

Comment:
If you should redo the above, we can possibly improve this as well and then smoothly lead to the list below. 

I think this sentence does not makes much sense: "While the strategies introduced above are usually enough to solve most conflicts" - the strategies will either ALWAYS succeed or always FAIL. Using fail-on-conflict alone will never _be enough_ 😄 

I think the argument should be: "If the resolution result is not satisfying (e.g. version picked is too high) or it fails (because you use fail-on-conflict) you have the following possibilities to fix it:"

Reply:
done